poi.config.js

```diff
@@ -85,7 +85,9 @@ module.exports = {
   },
   envs: {
     API_PATH: '/api/v1/',
-    AUTH_JWT_SCHEME
+    AUTH_JWT_SCHEME,
+    // The key to store the Vuex state in the storage
+    VUEX_STORAGE_KEY: 'TAILOR_APP_STATE'
```

Comment:
Personally I would go without that comment because now it is pretty obvious what is going on here 🙃

Reply:
Resolved